internal/initializer/crds.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package initializer
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/crossplane/crossplane-runtime/pkg/parser"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+)
+
+// NewCoreCRDs returns a new *CoreCRDs.
+func NewCoreCRDs(path string, s *runtime.Scheme) *CoreCRDs {
+	return &CoreCRDs{Path: path, Scheme: s}
+}
+
+// CoreCRDs makes sure the CRDs are installed.
+type CoreCRDs struct {
+	Path   string
+	Scheme *runtime.Scheme
+}
+
+// Run applies all CRDs in the given directory.
+func (c *CoreCRDs) Run(ctx context.Context, kube client.Client) error { // nolint:gocyclo
```

Comment:
Is this nolint comment needed here? The method doesn't look super complex.

Reply:
I guess it's left there because of the earlier implementation, removed.